uktrade/data-hub-frontend
# Change land date filters to month-date format.

## Description of change

This changes land date filters on investment collection page to be in month year format.

Note: this may not work in Firefox, but we are okay with that.

## Test instructions

Visit investment collection page and unroll Land date details section to see updated filters.

## Screenshots
### Before

![land dates before](https://user-images.githubusercontent.com/5889630/166902760-59abed58-9fe6-4351-bd95-007ed97afa32.png)

![land date before labels](https://user-images.githubusercontent.com/5889630/166903562-42335e0f-ddda-45b3-883d-aca331133c54.png)


### After

![land dates](https://user-images.githubusercontent.com/5889630/166902449-8e3ab232-5ad5-485c-905c-b16fce74eb25.png)

![land dates after labels](https://user-images.githubusercontent.com/5889630/166903586-9e94645d-3959-448d-a37f-8cd87059d2a5.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
